Forte.EpiServer.AzureSearch/ContentExtractor/IndexableContentExtractor.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using EPiServer;
+using EPiServer.Core;
+using Forte.EpiServer.AzureSearch.Extensions;
+using Forte.EpiServer.AzureSearch.Model;
+
+namespace Forte.EpiServer.AzureSearch.ContentExtractor
+{
+    public class IndexableContentExtractor : IContentExtractor
+    {
+        private readonly IContentLoader _contentLoader;
+
+        public IndexableContentExtractor(IContentLoader contentLoader)
+        {
+            _contentLoader = contentLoader;
+        }
+        public bool CanExtract(IContentData content)
+        {
+            return true;
+        }
+
+        public ContentExtractionResult Extract(IContentData content, ContentExtractorController extractor)
+        {
+            var stringValues = new List<string>();
+
+            var properties = content.GetIndexableProperties();
+
+            foreach (var property in properties)
+            {
+                switch (property.Value)
+                {
+                    case XhtmlString xhtmlString:
+                        stringValues.Add(XhtmlStringExtractor.GetPlainTextContent(xhtmlString, extractor));
+                        break;
+                    case BlockData localBlock:
+                        stringValues.AddRange(extractor.Extract(localBlock));
+                        break;
+                    //TODO: check if this case is used ORM blockData
+                    case ContentReference contentReference:
+                    {
+                        var propertyContent = _contentLoader.Get<IContent>(contentReference);
+
+                        if (propertyContent is BlockData == false)
+                        {
+                            continue;
```

Comment:
maybe it would be clearer to extract entire switch statement to separate method?
It's tricky to understand intention of break and continue. `Continue `referes to `foreach`, but `break `referes to switch (even though it could be valid for foreach)

maybe 

```
   foreach (var property in properties)
            {
                  stringValues.AddRange(ExtractStringValues(property))
           }
IReadOnlyCollection<string> ExtractStringValues(....){
                  switch (property.Value)
                {
                    case XhtmlString xhtmlString:
                       return new []{XhtmlStringExtractor.GetPlainTextContent(xhtmlString, extractor))};
}

```